tests/test_sql.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from pydo.config.config import default_cf_path
+from pydo.database.database_factory import DatabaseFactory
+from pydo.database.database import Database
+
+
+class TestSqlInit:
+
+    def test_init_sql(self):
+        """
+        initialise a new sql database
+        """
+
+        db: Database = DatabaseFactory.database(db_type='sqlite',
+                                                db_name='test_sql',
+                                                db_path='.')
+        db.initialise()
+        db_full_path = os.path.join('.', 'test_sql.db')
+        assert os.path.exists(db_full_path)
```

Comment:
You should use a proper test runner for tests, which will involve using the `self.assertXXX` functions instead of the `assert` keyword: https://docs.python.org/3/library/unittest.html